plugins/trader/portfolioManager.js

```diff
@@ -99,6 +110,14 @@ Manager.prototype.setPortfolio = function(callback) {
 
     this.portfolio = portfolio;
 
+    if(this.currencyAmountAvailableToTrade !== false && this.getBalance(this.currency) < this.currencyAmountAvailableToTrade) {
+      // reduce currencyAmountAvailableToTrade to equal the current balance if the current balance is lower than the configured currencyAmountAvailableToTrade
+      if(this.getBalance(this.currency) < 0)
+        this.currencyAmountAvailableToTrade = 0;
```

Comment:
@Ali1 I've been running this PR live, and it makes a successful first buy, though it's unable to finish a second trade if one of the orders wasn't filled, it seems as though the `currencyAmountAvailableToTrade ` is being set to 0 here and never changed back later. Is this the intended behavior?